### PR TITLE
Exposed FindTextDisplayEnd to imgui_internal.h

### PR DESCRIPTION
**Pull request by kylawl**

### CODE REVIEW

File: imgui_internal.h [hunk 1]
```diff
@@ -683,6 +683,8 @@ namespace ImGui
     IMGUI_API void          SetHoveredID(ImGuiID id);
     IMGUI_API void          KeepAliveID(ImGuiID id);
 
+    const char*             FindTextDisplayEnd(const char* text, const char* text_end = NULL);
```
Comment (ocornut) [inline imgui_internal.h]: Missing IMGUI_API here.

File: imgui.cpp [hunk 2]
```diff
@@ -2603,7 +2602,7 @@ void ImGui::RenderText(ImVec2 pos, const char* text, const char* text_end, bool
     const char* text_display_end;
     if (hide_text_after_hash)
     {
-        text_display_end = FindTextDisplayEnd(text, text_end);
+        text_display_end = ImGui::FindTextDisplayEnd(text, text_end);
```
Comment (ocornut) [inline imgui.cpp]: Could you undo adding those ImGui:: prefix for functions that are already in the ImGui namespace?
LogRenderedText is pretty much the only function here that needs them.

File: imgui_internal.h (hunk 1 above)
Comment (kylawl) [inline imgui_internal.h]: Ah, right thanks.

**Comment by ocornut** [comment]
Thanks Kyle, added some comments and I would merge that.
Note that no guarantee are provided with imgui_internal.h

I suggest renaming the function to `FindRenderedTextEnd` and putting it at the end of the block where RenderText() is declared, along with the `// Find the optional ## from which we stop displaying text.` comment.

**Comment by ocornut** [comment]
Merged, thanks!

**Comment by kylawl** [comment]
Cheers!

**Pull request merged**